app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditActivityTest.kt

```diff
@@ -313,7 +314,12 @@ class ProfileEditActivityTest {
         .inRoot(isDialog())
         .perform(click())
       testCoroutineDispatchers.runCurrent()
-      intended(hasComponent(ProfileListActivity::class.java.name))
+      val isTablet = context.resources.configuration.smallestScreenWidthDp >= 600
```

Comment:
I don't think this will be a correct way to access whether its a tablet or not, we have few test cases where we do isTablet conditionally, you can find in some test classes.

Reply:
Ok I'll check

Reply:
Done, please check

Reply:
` val isTablet = context.resources.configuration.smallestScreenWidthDp >= 600` any use of it?

Reply:
no i'll remove it